README.md

```diff
@@ -1,2 +1,106 @@
 # TeamMemberAPI
 An example API for managing a team using Django Rest Framework
+
+
+## Project Setup  - Ubuntu 18.04 LTS
+Here are commands to install all of the dependencies you will need for this project.
+
+    apt-get update
+    apt-get install python3
+    apt-get install python-pip
+    apt-get install python3-dev default-libmysqlclient-dev
+    apt-get install mysql-server
+    
+It is recommended that you use a virtualenv for this environment if it is a shared environment:
+    
+    python3 -m venv ~/.virtualenvs/djangodev
+    source ~/.virtualenvs/djangodev/bin/activate
+    
+Then install the following python libraries in your python env:
+
+    pip install Django
+    pip install mysqlclient
+    pip install djangorestframework
+    pip install markdown
+    pip install django-filter
+
+Note: you will need to setup this project with python3 because this is a Django2 project
+and Django 2 requires python3
+    
+You can setup mysql with some additional commands:
+
+    sudo systemctl start mysql
+    sudo systemctl enable mysql
+    
+Then you can log into mysql:
+
+    sudo /usr/bin/mysql -u root -p
+    
+Then create the django database user and give it permissions so that it can create and destroy databases for tests:
+
+    mysql> CREATE USER django;
+    mysql> GRANT ALL PRIVILEGES ON *.* TO 'django'
+    
+Note: for production environments restrict the permission of django database user more than this.
+
+Create the database that django will use:
+
+    mysql> CREATE DATABASE djangodb
+    
+You will likely need to migrate your your database settings after setting this up:
+    
+    python manage.py makemigrations team_api_app
```

Comment:
I don't think this step is necessary to setup the project. Only the developer adding tables/fields needs to run `makemigrations` to generate the migration files in the repo. Other developers just need to run `migrate` to apply those migrations.